docs/README.md

```diff
@@ -24,32 +25,31 @@ bundle install
+1. Modify a `.html` or `.md` file and save your changes, and the browser should quickly auto-refresh.
 
 # How the project is structured
 
 The [docs](https://github.com/Expensify/App/tree/main/docs) folder will contain the following main folders:
-- *_layouts*: This contains HTML templates used by Jekyll to create pages in the static site. For now it has only one file ([default.html](https://github.com/Expensify/App/blob/main/docs/_layouts/default.html)) which will render the common HTML elements and the assets on every page. It also supports the use of the [liquid template language](https://shopify.github.io/liquid/).
-- *_includes*: Contains HTML content that can be reused on different pages. For now, it contains various html elements that are used to populate the left-hand-nav (LHN).
-- *assets*: it will contain the three sub-folders css, images, and js.
-  - The css folder can contain either .css or .sass files (where SASS files will be processed by Jekyll and it will generate a CSS file in an output folder also generated by Jekyll).
-  -  In order to organize the image files, we're going to have multiple sub-folders grouped by subject in the images folder.
-- *hubs*: it will contain two HTML files that will have the content of the `Request Money` and `Send Money` hub pages respectively.
-- *articles*: it will contain sub-folders for each stage and each of them will contain all the articles related to that stage or subsection.
-- *_site*: this will be the output folder that will contain the generated HTML files. This is the default name by Jekyll but can be changed.
-index.html: it will have the homepage content.
-
-More details about the Jekyll project structure can be found [here](https://jekyllrb.com/docs/structure/).
+
+- *_layouts*: Contains HTML templates used by Jekyll to create pages in the static site. For now it has only one file ([default.html](https://github.com/Expensify/App/blob/main/docs/_layouts/default.html)) which will render the common HTML elements and the assets on every page.
+- *_includes*: Contains HTML content that can be reused on different pages. [Includes](https://jekyllrb.com/docs/includes/) can accept parameters and be nested, so they are quite powerful.
+- *assets*: Contains the three sub-folders css, images, and js. The css folder can contain either .css or .sass files (where SASS files will be processed by Jekyll and it will generate a CSS file in an output folder also generated by Jekyll).
+- *hubs*: Contains hubs that display various related articles, optionally grouped by sub-topic within the hub.
+- *articles*: Contains sub-folders for each hub and each of them will contain all the articles related to that hub.
+- *_data*: Contains [custom global data](https://jekyllrb.com/docs/datafiles/), available anywhere via liquid with the syntax `site.data.<filename>`. Notably, there is a file called routes.yml in there which defines the structure of the app and metadata for its pages. 
```

Comment:
I think it can be handy to link the [routes.yml](https://github.com/Expensify/App/blob/55f2c73d8e3bb0efd0e2a985c05f25e87d28f3cc/docs/_data/routes.yml) here:

```suggestion
- *_data*: Contains [custom global data](https://jekyllrb.com/docs/datafiles/), available anywhere via liquid with the syntax `site.data.<filename>`. Notably, there is a file called [routes.yml](https://github.com/Expensify/App/blob/55f2c73d8e3bb0efd0e2a985c05f25e87d28f3cc/docs/_data/routes.yml) in there which defines the structure of the app and metadata for its pages. 
```